brotli.go

```diff
@@ -0,0 +1,22 @@
+package httpgzip
+
+import (
+	"io"
+	"net/http"
+)
+
+func (fs *fileServer) findBrotliFile(w http.ResponseWriter, r *http.Request, fpath string) io.ReadSeeker {
+	brotliFilePath := fpath + ".br"
+
+	if file, err := fs.root.Open(brotliFilePath); err == nil {
+		defer file.Close()
+
+		wHeader := w.Header()
+		w.Header().Set("Content-Encoding", "br")
+		wHeader.Add("Vary", r.Header.Get("Accept-Encoding"))
```

Comment:
I find it a bit weird that a method called `findBrotliFile` as a side-effect sets HTTP headers 🤔 Could we move that to where the file is served instead?